ilan/install_ihs

```diff
@@ -40,9 +42,18 @@ function install_packages() {
   repo=$2
   shift 2
   packages="$@"
-  echo $repo
-if [[ $repo == *"IHS"* ]]
+
+if [[ $packages == *"java"* ]]
```

Comment:
I don't think you'd need all of this if you just installed Java and IHS at the same time by putting them against the same repo in versions.csv.